UMN-AutoPackager/public/Get-UMNGlobalConfig.ps1

```diff
@@ -0,0 +1,30 @@
+function Get-Config {
+    <#
+    .SYNOPSIS
+        Gets all the global configurations and value for each from a JSON file used by the auto packager.
+    .DESCRIPTION
+        This command retrieves from a json file the values of various global configurations used as part of the AutoPackager. It requires a the full path and name of the JSON file.
+    .EXAMPLE
+        Get-Config -Path .\config.json
+        Gets the values of the config.json file
+    .PARAMETER Path
+        The full path and file name of the JSON file to get the config from
+    #>
```

Comment:
We should standardize around help being outside the function right above the delcaration.